### PR TITLE
[WIP] test(E2ETest):Add e2e test cases for model

### PR DESCRIPTION
**Pull request by liuyihuicaicloud**
**What this PR does / why we need it**:

Add your description
1. add e2e test cases of model to cover the use case 
2. test cases for harbor project and model, pending

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: testutil/integration/model_test.go [hunk 1]
```diff
@@ -18,73 +28,127 @@ import (
 var _ = Describe("Model Registry", func() {
 	const timeout = time.Second * 5
 	const interval = time.Second * 1
+	type Format string
+
+	const (
+		FormatSavedModel  Format = "SavedModel"
```
Comment (codeflitting) [inline testutil/integration/model_test.go:34]: You can refer to it directly here, right? There is no need to repeat the definition

https://github.com/kleveross/klever-model-registry/blob/a480cc39ecd041b164383e35799785e3433461a9/pkg/apis/modeljob/v1alpha1/modeljob_types.go#L124

Comment (liuyihuicaicloud) [reply testutil/integration/model_test.go:34]: OK, will be changed

**Comment by simon-cj** [comment]
the title of PR, please ref https://github.com/kleveross/klever-model-registry/blob/master/CONTRIBUTING.md

File: testutil/integration/model_test.go [hunk 2]
```diff
@@ -18,73 +28,127 @@ import (
...
+		FormatTorchScript Format = "TorchScript"
+		FormatGraphDef    Format = "GraphDef"
+		FormatTensorRT    Format = "TensorRT"
+		FormatSKLearn     Format = "SKLearn"
+		FormatXGBoost     Format = "XGBoost"
+		FormatMLflow      Format = "MLflow"
+		FormatMLlib       Format = "MLlib"
+	)
+
+	var ModelExtract = []struct {
```
Comment (codeflitting) [inline testutil/integration/model_test.go:50]:
```suggestion
	var modelExtract = []struct {
```
Lowercase  is needed when no external package reference

Comment (liuyihuicaicloud) [reply testutil/integration/model_test.go:32]: OK

**Pull request closed without merging**